components/DetailPageSidebar/detail-page-sidebar.scss

```diff
@@ -0,0 +1,85 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/layout';
+
+@include exports("detail-page-sidebar") {
+  .bx--detail-page-sidebar__container {
+    margin: 0 5% 0 0;
+    .details-description,
+    .tag-description {
+      font-size: 0.875rem;
```

Comment:
We generally standardize around using the `font-size: rem(14px)` style of declaring font-size (for readability)